sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -168,7 +168,18 @@ class __SYCL_EXPORT modifiable_command_graph {
   /// executing.
   bool end_recording(const std::vector<queue> &RecordingQueues);
 
-protected:
+  /// Prints contents of graph to text file in DOT format
+  /// @param GraphName is a string appended to the output file name
+  void printGraphAsDot(const char *GraphName);
+
+  /// Checks is the graph_impl of Graph has a similar structure to
+  /// the graph_impl of the caller.
```

Comment:
Defining what a "similar structure" is here would make it a more informative comment.